### DISCUSSION TITLE
Provide q-field--disabled for disabled fields

### DISCUSSION THREAD

**Discussion by ajmas** [quasarframework/quasar#12556] [Ideas / Proposals] [answered]
To facilitate adjusting styles of disabled `q-input` and `q-select`, it would be useful to have `q-field--disabled` added to the component, when `:disabled="true"`, much in the same way as `q-field--readonly`.

For example, it would allow for the CSS customisation below to grey out a disabled field:

```css
.q-field--disabled {
  background: #eee;
}
```

As of now, I can't see how we could achieve the equivalent style adjustment without the addition of the proposed class.

**Comment by MilosPaunovic** [comment]
There actually is exact class you're requesting:
[https://codepen.io/milospaunovic/pen/qBVYomY?editors=1100](https://codepen.io/milospaunovic/pen/qBVYomY?editors=1100)

**Comment by rstoenescu** [comment, marked as answer]
The prop is "disable". Notice no ending with a "d". And it adds a specific class.

**Comment by ajmas** [comment]
You're right. I didn't notice, since `disabled` does kinda have an effect, since it appears to be passed onto the native field, even if it doesn't apply to the parent. Looking at the HTML indicates that.
